components/authentication/base/rh-idp/group-sync/konflux-rover-groups.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: GroupSync
+metadata:
+  name: konflux-rover-groups
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  providers:
+    - ldap:
```

Comment:
@gbenhaim Another question, should we prune the groups that are deleted from rover?

Reply:
yes

Reply:
Done in https://github.com/redhat-appstudio/infra-deployments/pull/3187